src/ethereum/curve/contracts.ts

```diff
@@ -5,6 +5,9 @@ import poolTokenV2Abi from "./abi/CurvePoolTokenV2.json";
 import poolTokenV3Abi from "./abi/CurvePoolTokenV3.json";
 import zapAbi from "./abi/CurveZap.json";
 import metaZapAbi from "./abi/CurveMetaZap.json";
+import metapoolFactory from "./abi/CurveMetapoolFactory.json";
```

Comment:
Can we have format? like this:
```suggestion
import CurveMetapoolFactoryAbi from "./abi/CurveMetapoolFactory.json";
```

to maintain uniformity with lines 14-21.

cc: @dhruvinparikh

Reply:
We should format previous lines as well, right?

Reply:
Yes, right..